### ISSUE TITLE
Queues get stuck often

### ISSUE THREAD

**Issue by markus-nclose** [ail-project/ail-framework#145]
Hi All, 
I have  fresh install - second one with the same issue. Installed using provided script.

OS: Ubuntu 20.04 server - updated to latest patch
Browser: Chrome 
```
Current Branch | master
Current Commit ID | df4bb531d216828878d0b463a5c80c974721ffd7
Current Tag | v4.1
```

The queues often goes in to stuck mode, upon inspection in the screen sessions, you can see Redis complains about missmatch in data types. The error message sometimes changes depending on the data types.

```
Traceback (most recent call last):
  File "/opt/ail-framework/bin/modules/abstract_module.py", line 102, in run
    self.compute(message)
  File "./Decoder.py", line 126, in compute
    content = self.decode_string(content, message, date, encoded_list, decoder['name'], decoder['encoded_min_size'])
  File "./Decoder.py", line 145, in decode_string
    Decoded.save_item_relationship(sha1_string, item_id)
  File "/opt/ail-framework/bin/lib/Decoded.py", line 240, in save_item_relationship
    r_serv_metadata.zincrby('hash_date:{}'.format(item_date), sha1_string, 1)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/commands/core.py", line 4064, in zincrby
    return self.execute_command("ZINCRBY", name, amount, value)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/client.py", line 1227, in execute_command
    return conn.retry.call_with_retry(
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/retry.py", line 46, in call_with_retry
    return do()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/client.py", line 1228, in <lambda>
    lambda: self._send_command_parse_response(
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/client.py", line 1204, in _send_command_parse_response
    return self.parse_response(conn, command_name, **options)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/client.py", line 1243, in parse_response
    response = connection.read_response()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/connection.py", line 842, in read_response
    raise response
redis.exceptions.ResponseError: value is not a valid float or out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./Decoder.py", line 174, in <module>
    module.run()
  File "/opt/ail-framework/bin/modules/abstract_module.py", line 107, in run
    self.redis_logger.critical(trace)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/pubsublogger/publisher.py", line 97, in critical
    log('CRITICAL', message)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/pubsublogger/publisher.py", line 62, in log
    redis_instance.publish(c, message)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/commands/core.py", line 5032, in publish
    return self.execute_command("PUBLISH", channel, message, **kwargs)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/client.py", line 1227, in execute_command
    return conn.retry.call_with_retry(
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/retry.py", line 46, in call_with_retry
    return do()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/client.py", line 1228, in <lambda>
    lambda: self._send_command_parse_response(
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/client.py", line 1203, in _send_command_parse_response
    conn.send_command(*args)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/connection.py", line 803, in send_command
    self.pack_command(*args), check_health=kwargs.get("check_health", True)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/connection.py", line 861, in pack_command
    for arg in map(self.encoder.encode, args):
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/redis/connection.py", line 110, in encode
    raise DataError(
redis.exceptions.DataError: Invalid input of type: 'list'. Convert to a bytes, string, int or float first.
```

![image](https://user-images.githubusercontent.com/38457858/169307107-e6251451-cbe5-424a-9287-84cbc7585734.png)

Any suggestions?

**Comment by markus-nclose** [comment]
Ubuntu doesn't seem like the right OS for it. I also couldn't view results in the Leaks Hunter. The correlation graphs also didn't work. 404

I installed Debian and all worked as expected.

**Comment by Terrtia** [comment]
HI @markus-nclose !
Thanks for the report !
The second exception is fixed with 512e61cadcd024487e3ab9d98155c3e9184a354e.
It seem like you are not using the required [redis-py](https://github.com/ail-project/ail-framework/blob/master/requirements.txt#L8) version.
